### PR TITLE
frees-http integrations

### PR DESCRIPTION
**Pull request by juanpedromoreno**
This PR migrates https://github.com/frees-io/freestyle-http to the main repository for the sake of simplicity in terms of maintainability and deployment.

At this moment, we are blocked until the new release of hammock: https://github.com/pepegar/hammock/issues/56

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -274,6 +277,19 @@ lazy val httpPlay = jvmModule("play", subFolder = Some("integrations/http"))
     ) ++ commonDeps
   )
 
+lazy val httpClient = module("http-client", subFolder = Some("integrations/http"))
+  .dependsOn(core)
+  .settings(resolvers += Resolver.jcenterRepo)
+  .jsSettings(sharedJsSettings: _*)
+  .crossDepSettings(
+    commonDeps ++ Seq(
+      %("hammock-core", "0.6.5-SNAPSHOT"),
```
Comment (juanpedromoreno) [inline build.sbt]: To be changed once the new version of Hammock is released.

**Comment by juanpedromoreno** [comment]
The only pending work in this PR would be to bump up the hammock version, once this is released.

The http-client integration code has been upgraded to the latest according to https://github.com/pepegar/hammock/commit/e3386c3da200cb1262ce43aef763b9e0a616d3b5.

**Comment by pepegar** [comment]
I will prioritize the release of Hammock now that I'm back. Sorry for blocking this :)

**Comment by juanpedromoreno** [comment]
Thanks @pepegar !

**Review by raulraja — APPROVED** [review]

**Comment by pepegar** [comment]
Great to see this merged! I will start working on the docs for the HTTP client. Also, I think we can delete the HTTP repo now?

**Comment by juanpedromoreno** [comment]
Sounds great.

Yep, @anamariamv is working archiving https://github.com/frees-io/freestyle-integrations, https://github.com/frees-io/freestyle-docs, https://github.com/frees-io/freestyle-examples and https://github.com/frees-io/freestyle-http, since all of them have been moved to this repo.

Thanks @pepegar !

**Comment by pepegar** [comment]
Awesome!

**Pull request merged**